### PR TITLE
[Azs.Backup.Admin] regenerate SDK according to updated spec

### PR DESCRIPTION
**Pull request by daozha**
## Description
Regenerate AZURESTACK BackupAdmin SDK according to spec change:
https://github.com/Azure/azure-rest-api-specs/pull/3225

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [x] Please add REST spec PR link to the SDK PR
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [ ] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [ ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Comment by deathly809** [comment]
@daozha The build failed, but it does not look like your changes are the cause.

**Review by dsgouda — CHANGES_REQUESTED** [review]
Since a package for the SDK was published earlier, the package version needs to be updated in the csproj and the assemblyinfo files.

**Comment by daozha** [comment]
@deathly809 Could you please let me know where you get the "build failed" information? All checks for this PR have passed and I guess build should be part of checks?

File: src/AzureStack/Admin/BackupAdmin/Backup.Admin/Microsoft.AzureStack.Management.Backup.Admin.csproj [hunk 1]
```diff
@@ -7,7 +7,7 @@
     <PropertyGroup>
         <PackageId>Microsoft.AzureStack.Management.Backup.Admin</PackageId>
         <Description>Provides developers with libraries for the Backup admin under AzureStack to manage backups.</Description>
-        <VersionPrefix>0.1.0-preview</VersionPrefix>
+        <VersionPrefix>0.1.1-preview</VersionPrefix>
         <AssemblyName>Microsoft.AzureStack.Management.Backup.Admin</AssemblyName>
         <PackageTags>Microsoft Azure Stack;Backup;REST HTTP client;azureostackfficial;netcore451511</PackageTags>
         <PackageReleaseNotes>
```
Comment (dsgouda) [inline src/AzureStack/Admin/BackupAdmin/Backup.Admin/Microsoft.AzureStack.Management.Backup.Admin.csproj]: Please update `PackageReleaseNotes`
Changing return types is a breaking change, please update version to `0.2.0-preview`

File: src/AzureStack/Admin/BackupAdmin/Backup.Admin/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -14,5 +14,5 @@
 [assembly: AssemblyCulture("")]
 [assembly: NeutralResourcesLanguage("en")]
 
-[assembly: AssemblyVersion("0.1.0.0")]
-[assembly: AssemblyFileVersion("0.1.0.0")]
+[assembly: AssemblyVersion("0.1.1.0")]
```
Comment (dsgouda) [inline src/AzureStack/Admin/BackupAdmin/Backup.Admin/Properties/AssemblyInfo.cs]: Please do not change the `AssemblyVersion` here, only `AssemblyFileVersion` needs to be updated

**Review by dsgouda — CHANGES_REQUESTED** [review]

File: src/AzureStack/Admin/BackupAdmin/Backup.Admin/Microsoft.AzureStack.Management.Backup.Admin.csproj [hunk 3]
```diff
@@ -16,6 +16,9 @@
                 1) Configure where backups are stored
                 2) Perform backups 
                 3) List and restore completed backup. 
+                
+                0.2.0-preview breaking change:
```
Comment (dsgouda) [inline src/AzureStack/Admin/BackupAdmin/Backup.Admin/Microsoft.AzureStack.Management.Backup.Admin.csproj]: Replace the first three points with this text

Comment (daozha) [reply src/AzureStack/Admin/BackupAdmin/Backup.Admin/Microsoft.AzureStack.Management.Backup.Admin.csproj]: Sure. Thanks.

**Review by dsgouda — APPROVED** [review]
LGTM

**Comment by dsgouda** [comment]
Will merge on CIs passing

**Comment by daozha** [comment]
Thanks!

**Pull request merged**